tsconfig.base.json

```diff
@@ -16,6 +16,7 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@monorepo/shared/ui-library": ["libs/shared/ui-library/src/index.ts"]
```

Comment:
What is suggested by Nx as a best practice is to use the directory + library name as a path like `@shared/ui-library`.

Reply:
this was generated and haven't thought too much about that will update `@monorepo/shared/ui-library` into `/shared/ui-library` then!